### PR TITLE
Added Chrome specific text to onboarding page

### PR DESCRIPTION
**Pull request by CatastrophiClam**
Tested on Chrome and Firefox on Windows

### CODE REVIEW

File: server/templates/onboarding_page.html [hunk 1]
```diff
@@ -27,11 +27,19 @@ <h3>Step 1 — Download your transcript</h3>
               Quest</a> and follow the instructions <a target="_blank" href="https://uwaterloo.ca/quest/help/students/how-do-i/unofficial-transcript">here</a> to download your transcript as a PDF.
           </p>
           <br/>
-          <h3>Step 2 — Open and copy your transcript</h3>
+          <h3>Step 2 — Open and copy your transcript in Google Chrome</h3>
```
Comment (dshynkev) [inline server/templates/onboarding_page.html]: This should probably be parametrized by Chrome/not Chrome too if you want to change it.

Comment (AyushK1) [reply server/templates/onboarding_page.html]: @CatastrophiClam

File: server/templates/onboarding_page.html [hunk 2]
```diff
@@ -27,11 +27,20 @@ <h3>Step 1 — Download your transcript</h3>
               Quest</a> and follow the instructions <a target="_blank" href="https://uwaterloo.ca/quest/help/students/how-do-i/unofficial-transcript">here</a> to download your transcript as a PDF.
           </p>
           <br/>
-          <h3>Step 2 — Open and copy your transcript</h3>
+          {% if not ("Chrome" in user_agent or "chrome" in
+          user_agent) %}
+	  <h3>Step 2 - Open and copy your transcrip</h3>
```
Comment (AyushK1) [inline server/templates/onboarding_page.html]: Amend `transcrip` to `transcript` @CatastrophiClam

**Comment by derrekchow** [comment]
Tested on Chrome and Safari on Mac

**Comment by AyushK1** [comment]
LGTM 👍 ! Awesome work :)

**Pull request merged**